run_acn_node_standalone.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please explain where this file was taken from and what the diff is (exact links)

Reply:
[original Dockerfile](https://github.com/valory-xyz/open-aea/blob/main/scripts/acn/Dockerfile)

diff: 
```
# utils
-RUN apt install -y wget
+RUN apt install -y wget subversion
 
 # golang
-RUN wget https://dl.google.com/go/go1.17.7.linux-amd64.tar.gz && \
-  tar -xzvf go1.17.7.linux-amd64.tar.gz -C /usr/local && \
+RUN wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz && \
+  tar -xzvf go1.13.8.linux-amd64.tar.gz -C /usr/local && \
   export PATH=$PATH:/usr/local/go/bin && echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && \
   mkdir $HOME/go
 
@@ -23,7 +23,7 @@ RUN python3 -m pip install pymultihash ecdsa base58
 WORKDIR /acn/
 
 # get node source code
-COPY . /acn/node
+RUN svn export https://github.com/valory-xyz/open-aea/trunk/packages/open_aea/connections/p2p_libp2p/libp2p_node/ /acn/node
```